application/org.openjdk.jmc.flightrecorder.flamegraph/build.properties

```diff
@@ -1,6 +1,4 @@
-source.. = src/main/java/,\
-           src/main/js/,\
-           src/main/resources/
+source.. = src/main/java/
```

Comment:
Any particular reason why src/main/resources is missing here now?

Reply:
I'm not sure, I never manually touched that file. It's odd indeed I'll add it back.